Moving the work from the staging 'branch' to an actual branch of Odoo.

jeffmiller00 opened the pull request:
These are the changes that @smhassandev did in the https://github.com/linenmaster/infinite-laundry-crm/tree/staging branch applied to Odoo.

Commit:
Moving the work from the staging 'branch' to an actual branch of Odoo.

Commit:
customization

smhassandev commented:
Hello Jeff! Did you test the work?

On Tue, Aug 20, 2024 at 9:09 PM Jeff Miller ***@***.***>
wrote:

> These are the changes that @smhassandev <https://github.com/smhassandev>
> did in the
> https://github.com/linenmaster/infinite-laundry-crm/tree/staging branch
> applied to Odoo.
> ------------------------------
> You can view, comment on, or merge this pull request online at:
>
>   https://github.com/linenmaster/infinite-laundry-crm/pull/1
> Commit Summary
>
>    - 2fe6f32
>    <https://github.com/linenmaster/infinite-laundry-crm/pull/1/commits/2fe6f32f75dc1fcbf2511878d33d7c8c7ab2edcf>
>    Moving the work from the staging 'branch' to an actual branch of Odoo.
>
> File Changes
>
> (21 files
> <https://github.com/linenmaster/infinite-laundry-crm/pull/1/files>)
>
>    - *A* cyb_crm_api/__init__.py
>    <https://github.com/linenmaster/infinite-laundry-crm/pull/1/files#diff-a89152a79c89bc958438a8fddde415ac29227ad85cdc7ab8baf51fc46d28ef5f>
>    (4)
>    - *A* cyb_crm_api/__manifest__.py
>    <https://github.com/linenmaster/infinite-laundry-crm/pull/1/files#diff-c03e8645df40c5086db3f7fabef442aefbe314e45cbcac592376ebc6356efa2b>
>    (35)
>    - *A* cyb_crm_api/controllers/__init__.py
>    <https://github.com/linenmaster/infinite-laundry-crm/pull/1/files#diff-6d661a924c3bf634238a26035458704eb4633bd3f1e0c838dc80190c4188017c>
>    (3)
>    - *A* cyb_crm_api/controllers/controllers.py
>    <https://github.com/linenmaster/infinite-laundry-crm/pull/1/files#diff-1cfeac7e5e4d6ca438224f3e475c9a7696e587877daeb81b1b59283c652b3f82>
>    (37)
>    - *A* cyb_crm_api/demo/demo.xml
>    <https://github.com/linenmaster/infinite-laundry-crm/pull/1/files#diff-15af69604535327c59dea8f7fea1d9418912002ebc2edd95b083793ef0154468>
>    (30)
>    - *A* cyb_crm_api/models/__init__.py
>    <https://github.com/linenmaster/infinite-laundry-crm/pull/1/files#diff-15bf4f28e31ebac0dd86480c72bda7b25caa10058f5ee7054751b96b470f2569>
>    (3)
>    - *A* cyb_crm_api/models/models.py
>    <https://github.com/linenmaster/infinite-laundry-crm/pull/1/files#diff-06d2a1869b8c1624f7852c8eea2362d0981effa0a62df34b29aee967df7ae06a>
>    (4)
>    - *A* cyb_crm_api/security/ir.model.access.csv
>    <https://github.com/linenmaster/infinite-laundry-crm/pull/1/files#diff-3a4e873206ce19f789503d6620d820efdc2e3ba771ffca8f8a63aae893e89017>
>    (2)
>    - *A* cyb_crm_api/views/templates.xml
>    <https://github.com/linenmaster/infinite-laundry-crm/pull/1/files#diff-425fc7eb2a1c03dac4e3a9cb7a813aed0943360c91b18199bea8ce5f03eaa051>
>    (24)
>    - *A* cyb_crm_api/views/views.xml
>    <https://github.com/linenmaster/infinite-laundry-crm/pull/1/files#diff-134848d70246f8e117a601f3d4b9af5dafdaebb82e6619a10999fad0ffde7f77>
>    (60)
>    - *A* cyb_web_customization/__init__.py
>    <https://github.com/linenmaster/infinite-laundry-crm/pull/1/files#diff-967e579a640c941beaaa89ae84fbd720406c37fcb9584a5b339c7bc2930ebd75>
>    (1)
>    - *A* cyb_web_customization/__manifest__.py
>    <https://github.com/linenmaster/infinite-laundry-crm/pull/1/files#diff-fcc58485c5bb24bdcd22734672c64f405c3484d767614671242345012ffd923a>
>    (26)
>    - *A* cyb_web_customization/models/__init__.py
>    <https://github.com/linenmaster/infinite-laundry-crm/pull/1/files#diff-a17089c06f3299047d82856801dc16fbab95cfd059f18fb0faa7f4bc734c0833>
>    (1)
>    - *A* cyb_web_customization/models/models.py
>    <https://github.com/linenmaster/infinite-laundry-crm/pull/1/files#diff-f7a0c84aae2d8f80c326cd2d51f39afff355ce787f27571b7af855848fb4295b>
>    (1)
>    - *A* cyb_web_customization/static/description/icon.jpg
>    <https://github.com/linenmaster/infinite-laundry-crm/pull/1/files#diff-eca9e5db536d904754b23b2fa392b383992abcd65be6ed7b95ec5c00d4fdb17e>
>    (0)
>    - *A* cyb_web_customization/static/description/lg.jpg
>    <https://github.com/linenmaster/infinite-laundry-crm/pull/1/files#diff-9eae6ec788501e373940e0d9cae3cdc0537516d7501da80781c323e1bd82f7ca>
>    (0)
>    - *A* cyb_web_customization/static/src/js/cyb_user_men_items.js
>    <https://github.com/linenmaster/infinite-laundry-crm/pull/1/files#diff-924c34f65eb3634d85828c8e2f01727f7caec4a4ad1304adfba174a9bb64aef9>
>    (8)
>    - *A* cyb_web_customization/static/src/scss/css_variables.scss
>    <https://github.com/linenmaster/infinite-laundry-crm/pull/1/files#diff-163c4d643a5ea8f957bf081d263cf1c4fd11e36a1e32181532fe2bf1212302f0>
>    (303)
>    - *A* cyb_web_customization/static/src/xml/template.xml
>    <https://github.com/linenmaster/infinite-laundry-crm/pull/1/files#diff-3c75083cc1f2393d20c8b44d7af905bb3f10beedba8167de322756c1bea52195>
>    (40)
>    - *A* cyb_web_customization/views/templates_view.xml
>    <https://github.com/linenmaster/infinite-laundry-crm/pull/1/files#diff-157f1f56b9ddf7ffa0a3750f5403821c72af102c62bcb417d7d82b0d58e533c1>
>    (16)
>    - *A* cyb_web_customization/views/views.xml
>    <https://github.com/linenmaster/infinite-laundry-crm/pull/1/files#diff-56aca9068cd8e412fe4b591af3caecfdee60d87baa497ca8418b75e286c18edb>
>    (53)
>
> Patch Links:
>
>    - https://github.com/linenmaster/infinite-laundry-crm/pull/1.patch
>    - https://github.com/linenmaster/infinite-laundry-crm/pull/1.diff
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/linenmaster/infinite-laundry-crm/pull/1>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BBK6USWYYJZ657IRRP57GV3ZSNS5HAVCNFSM6AAAAABM2GJXVKVHI2DSMVQWIX3LMV43ASLTON2WKOZSGQ3TKOJYGU4TAOI>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

jeffmiller00 commented on addons/cyb_web_customization/views/views.xml:
```diff
@@ -0,0 +1,53 @@
+                <div t-attf-class="card border-0 mx-auto bg-100 {{login_card_classes}} o_database_list" style="max-width: 300px;">
+                    <div class="card-body">
+                        <div t-attf-class="text-center pb-3 border-bottom {{'mb-3' if form_small else 'mb-4'}}">
+                            <img t-attf-src="/web/binary/company_logo{{ '?dbname='+db if db else '' }}" alt="Logo" style="max-height:120px; max-width: 100%; width:auto"/>
```
@smhassandev I don't see this file path any where...

jeffmiller00 commented on addons/cyb_web_customization/static/description/icon.jpg:
Where does this file show up?

jeffmiller00 commented on addons/cyb_web_customization/static/description/lg.jpg:
Where does this file show up?

jeffmiller00 commented:
@smhassandev I've added a number of comments in the PR. Also not all requirements are completed & I added a few more: https://linenmaster.sharepoint.com/:w:/s/Technology/ETQGBwz3DdZMvo_7lF8qNYsBLvKCuFBrMGuVPhvPunia1g?e=knfWca

Commit:
Move instructions from the Upwork chat to the repo.

jeffmiller00 commented on InfiniteLaundry CRM README.md:
```diff
@@ -0,0 +1,43 @@
+“Apps” and “Settings”
+Those are related to access rights, those will be handled through access rights.
+
+Note for data: We wouldn't load demo data while creating production db or if you want to load and remove after then you have to give me your odoo access or we can do in meeting.
```
Please add a section below here to cover the CRM API Word document.

jeffmiller00 commented on addons/cyb_web_customization/static/description/icon.jpg:
Where does this file show up?

jeffmiller00 commented on addons/cyb_web_customization/static/description/lg.jpg:
Where does this file show up?

smhassandev commented:
Can you please share the requirements documents in PDF form here? It is
giving an error when I try to open it. Please download in pdf form and
share it here.

On Thu, Aug 22, 2024 at 10:31 PM Jeff Miller ***@***.***>
wrote:

> ***@***.**** commented on this pull request.
> ------------------------------
>
> On addons/cyb_web_customization/static/description/lg.jpg
> <https://github.com/linenmaster/infinite-laundry-crm/pull/1#discussion_r1727549642>
> :
>
> Where does this file show up?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/linenmaster/infinite-laundry-crm/pull/1#pullrequestreview-2255252756>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BBK6USUNZACAGOVWBT5ZZKDZSYN7BAVCNFSM6AAAAABM2GJXVKVHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMZDENJVGI2TENZVGY>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

jeffmiller00 committed:
Moving the requirements document to PDF.

jeffmiller00 commented:
> Can you please share the requirements documents in PDF form here? It is giving an error when I try to open it. Please download in pdf form and share it here.
> […](#)
> On Thu, Aug 22, 2024 at 10:31 PM Jeff Miller ***@***.***> wrote: ***@***.**** commented on this pull request. ------------------------------ On addons/cyb_web_customization/static/description/lg.jpg <[#1 (comment)](https://github.com/linenmaster/infinite-laundry-crm/pull/1#discussion_r1727549642)> : Where does this file show up? — Reply to this email directly, view it on GitHub <[#1 (review)](https://github.com/linenmaster/infinite-laundry-crm/pull/1#pullrequestreview-2255252756)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/BBK6USUNZACAGOVWBT5ZZKDZSYN7BAVCNFSM6AAAAABM2GJXVKVHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMZDENJVGI2TENZVGY> . You are receiving this because you were mentioned.Message ID: ***@***.***>

I've committed the PDF requirements to the repo as Requirements.pdf and will keep it updated.

jeffmiller00 commented:
This was the first attempt at this work, but closing now.
